drivers/volume/portworx/portworx.go

```diff
@@ -854,6 +859,19 @@ func (p *portworx) SnapshotCreate(
 		request.Labels[cloudBackupOwnerLabel] = "stork"
 		p.addCloudsnapInfo(request, snap)
 
+		if value, present := snap.Metadata.Annotations[incrementalCountAnnotation]; present {
+			incrementalCount, err := strconv.ParseUint(value, 10, 32)
+			if err != nil {
+				return nil, nil, fmt.Errorf("invalid cloudsnap-incremental-count specified %v: %v", p, err)
+			}
+			if incrementalCount <= 0 {
```

Comment:
Good to have a check if this is < 0 as it's invalid input.

Reply:
<0 would mean to keep the default